### ISSUE TITLE
MT: Search Tests

### ISSUE THREAD

**Issue by mollyaljenkins** [aemsites/vg-macktrucks-com#46]
### Description
Once brands are migrated to the mono repos search tests are to be defined as below.
These are tests to check search functionalities are working.
Like we do tests when a new release is deployed, it would be beneficial to also add these quick tests over the search service

### Define tests for search:

**Three endpoints:**

- [ ] Search endpoint (search bar)
- [ ] Suggest endpoint (suggestions that appear when user types)
- [ ] Recommend endpoint (new magazine search endpoint where tags can be selected)
- [ ] Facets filterings
- [ ] Results response

### Out of Scope 

- [ ] Executing the tests

**Comment by Meli-lazarte** [comment]
This ticket is dependent on ticket #125.

**Comment by girishdigrajkar** [comment]
Please see attached test cases:

[MT.Search.Tests.46.Test.cases.xlsx](https://github.com/user-attachments/files/18209285/MT.Search.Tests.46.Test.cases.xlsx)


@netmanito, @jonatan-lledo-netcentric : Please see above TCs ( Applicability as per COM & other markets by using respective links )

**Comment by jonatan-lledo-netcentric** [comment]
since this is just a general list and no code was needed to be merged, I consider this as done and closed
